app/src/main/java/com/solux/flory/presentation/gift/GiftFragment.kt

```diff
@@ -19,7 +19,7 @@ class GiftFragment : BindingFragment<FragmentGiftBinding>(FragmentGiftBinding::i
 
     private fun leftArrowClick() {
         binding.ivGiftLeftArrow.setOnClickListener {
-            val intent = Intent(requireContext(), SearchneighborActivity::class.java)
+            val intent = Intent(requireContext(), SearchNeighborActivity::class.java)
             startActivity(intent)
```

Comment:
```suggestion
            Intent(requireContext(), SearchNeighborActivity::class.java).apply {
                startActivity(this)
            }
```
이런식으로 코드 작성하면 불필요한 변수 생성을 막을 수 있습니다